src/components/Chip/Chip.test.tsx

```diff
@@ -1,140 +1,155 @@
-import { shallow, mount } from "enzyme";
+import { render, screen, within } from "@testing-library/react";
 import React from "react";
 
-import Chip from "./Chip";
+import Chip, { Label } from "./Chip";
+import userEvent from "@testing-library/user-event";
 
 describe("Chip ", () => {
   it("renders default chip", () => {
-    const wrapper = shallow(<Chip value="positive" />);
-    expect(wrapper).toMatchSnapshot();
+    render(<Chip role="option" value="positive" />);
```

Comment:
Instead of passing arbitrary role just to use it in selector, maybe we can pass some test id? Or use `container` or `container.firstChild`.

Reply:
I used a testid for this one so that I didn't have to disable the node access and container rules in all the tests.